client.go

```diff
@@ -59,16 +60,22 @@ func (c *SpiceClient) Init(apiKey string) error {
 		return fmt.Errorf("error getting system cert pool: %w", err)
 	}
 
+	grpcDialOpts := []grpc.DialOption{grpc.WithDefaultCallOptions(
+		grpc.MaxCallRecvMsgSize(MAX_MESSAGE_SIZE_BYTES),
+		grpc.MaxCallSendMsgSize(MAX_MESSAGE_SIZE_BYTES))}
+
+	if strings.HasPrefix("grpc://", c.address) {
+		grpcDialOpts = append(grpcDialOpts, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	} else {
+		grpcDialOpts = append(grpcDialOpts, grpc.WithTransportCredentials(credentials.NewClientTLSFromCert(systemCertPool, "")))
```

Comment:
does this option work with and without the "grpc+tls://" schema prefix?

Reply:
Yep!